src/components/ecl-date-block/ecl-date-block.stories.ts

```diff
@@ -1,36 +1,49 @@
 const getArgs = () => {
   return {
-    variant: 'default',
+    variant: 'ongoing',
     day: '26',
     month: 'Sep',
+    monthFull: 'September',
     year: '2022',
     dateTime: '2019-09-26',
```

Comment:
should we keep date-time?

Reply:
what do you mean..? It's part of the markup of the vanilla component: 
`{% set _date_time = date_time|default('') %}   {% if _date_time is not empty %}
    {% set _extra_attributes = _extra_attributes ~ ' datetime="' ~ _date_time|e('html_attr') ~ '"' %}
  {% endif %}
`

Reply:
in the control i was reffering, i saw that on v5-dev we dont have this control

Reply:
ah, true, but is not that we make a carbon copy version of the stories, in ECL sometimes the controls are requested by the designers, in this component we have that property, so why not having a control showing that.. ;)